webapp/src/components/assetEditor/assetSidebar.tsx

```diff
@@ -189,35 +201,42 @@ class AssetSidebarImpl extends React.Component<AssetSidebarProps, AssetSidebarSt
                     return <div key={el.name} className="asset-editor-sidebar-detail">{`${el.name}: ${el.value}`}</div>
                 })}
             </List>
-            { asset && <List className="asset-editor-sidebar-controls">
-                {canEdit && <Button
-                    label={lf("Edit")}
-                    title={lf("Edit the selected asset")}
-                    ariaLabel={lf("Edit the selected asset")}
-                    leftIcon="icon edit"
-                    className="asset-editor-button"
-                    onClick={this.editAssetHandler} />}
-                <Button
-                    label={lf("Duplicate")}
-                    title={lf("Duplicate the selected asset")}
-                    ariaLabel={lf("Duplicate the selected asset")}
-                    leftIcon="icon copy"
-                    className="asset-editor-button"
-                    onClick={this.duplicateAssetHandler} />
-                {canCopy && <Button
-                    label={lf("Copy")}
-                    title={lf("Copy the selected asset to the clipboard")}
-                    ariaLabel={lf("Copy the selected asset to the clipboard")}
-                    leftIcon="icon paste"
-                    className="asset-editor-button"
-                    onClick={this.copyAssetHandler} />}
-                {canDelete && <Button
-                    label={lf("Delete Asset")}
-                    title={lf("Delete the selected asset")}
-                    ariaLabel={lf("Delete the selected asset")}
-                    className="delete-asset"
-                    onClick={this.showDeleteModal} />}
-            </List>}
+            <List className="asset-editor-sidebar-controls">
+                    { asset && canEdit && <Button
```

Comment:
```suggestion
                { asset && canEdit && <Button
```
nit: this section is double indented until the asset palette button; unindenting will also remove a lot of the diff which is nice :)

Reply:
nice catch! thanks